sequelizeDatabase/modelSetup.js

```diff
@@ -2,6 +2,7 @@ const Sequelize = require('sequelize');
 let result = require("dotenv").config();
 
 const sequelize = new Sequelize(process.env.DATABASE,process.env.USER,process.env.PASSWORD,{
+    logging: false,
```

Comment:
Aaron added this to keep the log from being filled with the SQL statements as he was debugging.
**NOTE:** This line may be changed in the future or removed entirely for other debugging purposes.